MeetingView.js

```diff
@@ -38,6 +39,14 @@ export default class MeetingView extends Component {
     };
   }
 
+  componentWillMount() {
+    IdleTimerManager.setIdleTimerDisabled(true);
```

Comment:
This call only handles iOS right now. I suppose we can make a platform specific view or we can extend this bridge to support android.